db/system_distributed_keyspace.cc

```diff
@@ -237,27 +237,15 @@ future<> system_distributed_keyspace::remove_view(sstring ks_name, sstring view_
             false).discard_result();
 }
 
-/* We want to make sure that writes/reads to/from cdc_generations and cdc_streams
+/* We want to make sure that writes/reads to/from CDC management-related distributed tables
```

Comment:
I think we're losing some info here. Previously it was clear which tables we're talking about. Now it's not clear whether the comment is about user facing tables or the internal tables.

Reply:
Very easy to check by grepping quorum_if_many in this file.

Reply:
I agree it was more useful to have the actual list here (i.e., denormalization!), but it's not useful to have a wrong list :-)